mlflow/store/model_registry/abstract_store.py

```diff
@@ -334,27 +334,28 @@ def copy_model_version(self, src_mv, dst_name):
         :return: Single :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
                  the cloned model version.
         """
-        raise MlflowException(
-            "Method 'copy_model_version' has not yet been implemented for the current model "
-            "registry backend. To request support for implementing this method with this backend, "
-            "please submit an issue on GitHub."
-        )
-
-    def _copy_model_version_impl(self, src_mv, dst_name):
         try:
             self.create_registered_model(dst_name)
         except MlflowException as e:
             if e.error_code != ErrorCode.Name(RESOURCE_ALREADY_EXISTS):
                 raise
 
-        return self.create_model_version(
-            name=dst_name,
-            source=f"models:/{src_mv.name}/{src_mv.version}",
-            run_id=src_mv.run_id,
-            tags=[ModelVersionTag(k, v) for k, v in src_mv.tags.items()],
-            run_link=src_mv.run_link,
-            description=src_mv.description,
-        )
+        try:
+            mv_copy = self.create_model_version(
+                name=dst_name,
+                source=f"models:/{src_mv.name}/{src_mv.version}",
+                run_id=src_mv.run_id,
+                tags=[ModelVersionTag(k, v) for k, v in src_mv.tags.items()],
+                run_link=src_mv.run_link,
+                description=src_mv.description,
+            )
+        except MlflowException as e:
+            raise MlflowException(
+                f"Failed to create model version copy. The current model registry backend "
+                f"may not yet support model version URI sources. Error: {e}"
```

Comment:
Do we need to include the Error message in the main exception for readability and context for users or is the chained stack trace clear? (Asking out of curiosity here, not a requested change)

Reply:
Just personally, I often just read the main error and kind of ignore the upstream error, so having the upstream error in the main error would be helpful to me. It could just be me though... other developers might have much better habits to scroll up. What do you think?

Reply:
I usually base the decision on what critical information is included in the exception we're wrapping. If there is VERY useful info in the main message that means that no one but a dev doing troubleshooting is going to have to read through the stack trace, then it's usually a good idea. But if it's something like "An Error Occurred." then chaining is best. 
For this case, it makes sense to do both inline and chained (well, chained is required, IMHO) since we control the messages on both. 
For readability, we might want to introduce a newline \n prior to the `Error: {e}` statement.

Reply:
Sounds great!